src/components/shared/ArrowPanel.css

```diff
@@ -86,12 +86,18 @@
   z-index: -1;
 }
 
-.arrowPanelTitle {
+.arrowPanelTitle,
+.arrowPanelSubTitle {
   font-size: 1.2em;
   padding: 10px;
   margin: 0;
 }
 
+.arrowPanelSubTitle {
```

Comment:
I think this could use some margin-top as well. At the moment it is very squished against the previous line.